tools/manifest/sourcefile.py

```diff
@@ -479,7 +483,7 @@ def timeout(self):
         """The timeout of a test or reference file. "long" if the file has an extended timeout
         or None otherwise"""
         if self.script_metadata:
-            if any(m == (b"timeout", b"long") for m in self.script_metadata):
+            if any(m == ("timeout", "long") for m in self.script_metadata):
```

Comment:
Maybe add `u` prefix here and to other literals
```suggestion
            if any(m == (u"timeout", u"long") for m in self.script_metadata):
```

(Although I know `u"foo" == "foo"` in Python 2.)

And there's also `quic()` that contains the same logic.

Reply:
So I think I'd be happier looking at u-prefixing in a followup now that this one is passing tests, since this is basically an extended bustage fix. Maybe for files like this which are fully converted we should investigate using `unicode_literals` so we're getting consistent types.